docs/contributor_guidelines.rst

```diff
@@ -1,12 +1,27 @@
 Contributing Guidelines
 =======================
 
-Signing commits
+Signing Commits
```

Comment:
We're all over the place on title style, just keeping it consistent in this file.